src/main/java/top/seraphjack/simplelogin/server/capability/ISLEntry.java

```diff
@@ -18,4 +19,8 @@
 
     int getGameType();
 
+    Position getLastPosition();
```

Comment:
Add a default implementation to return `defaultPosition`?

Reply:
No. It must be implemented